packages/cozy-pouch-link/src/CozyPouchLink.spec.js

```diff
@@ -148,17 +164,38 @@ describe('CozyPouchLink', () => {
       expect(link.supportsOperation(operationTODO)).toBe(true)
     })
 
-    it('should forward if the doctype is sync only for read access', async () => {
+    it('should forward if the doctype is synched only for read access and the query is a mutation', async () => {
       await setup({
         doctypesReplicationOptions: {
           'io.cozy.todos': { strategy: 'fromRemote' }
         }
       })
-      const query = Q(TODO_DOCTYPE)
-      expect.assertions(1)
-      await link.request(query, null, () => {
-        expect(true).toBe(true)
+      link.pouches.isSynced = jest.fn().mockReturnValue(true)
+      link.request(
```

Comment:
Nit: I am a bit surprised that there is not await here ? I think the `forward` argument should be async, so that we can put an await here.

Reply:
Since we can pass what we want in the forward argument, I do not need to await because here I want to test that the forward argument is called and all the async work in `request` is done later. But let's add the await in order to be consistent